docs/3.1/docs/json-api/index.rst

```diff
@@ -649,7 +566,7 @@ Where:
 However, because ``contractId`` is always unambiguous, you may alternatively simply specify the interface ID as the ``templateId`` argument, and ignore ``choiceInterfaceId`` entirely.
 This isn't true of exercise-by-key or create-and-exercise, so we suggest treating this request as if this alternative isn't available.
 
-.. _exercise-response:
+.. _exercise-response: (using ``/v2/commands/submit-and-wait``)
```

Comment:
For exercise it would probably be more useful to show `submit-and-wait-for-transaction`, because the results are more analogous to what the old `/exercise` endpoing did, and you also get the choice's return value.